File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        answer = []
        i = 1
        for i in range(1, n + 1):
            fizz = ""
            if i % 3 == 0:
                fizz += "Fizz"
            if i % 5 == 0:
                fizz += "Buzz"
            if (i % 3 != 0 and i % 5 != 0):
                fizz = str(i)
            answer.append(fizz)
        return answer

File: 0412-fizz-buzz/0412-fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let ans = [];
    let fizz = ""
    for (let i = 1; i <= n; i++) {
        fizz = "";
        if (i%3 === 0) {
            fizz = "Fizz";
        }
        if (i%5 === 0) {
            fizz += "Buzz";
        }
        if (i%3 != 0 && i%5 != 0) {
            fizz = i.toString();
        }
        ans.push(fizz);
    }
    return ans;
};